frame/pablo/src/lib.rs

```diff
@@ -573,6 +634,107 @@ pub mod pallet {
 	}
 
 	impl<T: Config> Pallet<T> {
+		fn default_pblo_staking_pool_config(
+			pool_id: &T::PoolId,
+			pair: &CurrencyPair<T::AssetId>,
+		) -> Result<
+			RewardPoolConfiguration<
+				T::AccountId,
+				T::AssetId,
+				T::BlockNumber,
+				RewardConfigsOf<T>,
+				DurationPresets<T>,
+			>,
+			DispatchError,
+		> {
+			let mut rewards = BTreeMap::new();
+			let max_rewards: T::Balance = T::Convert::convert(MAX_REWARDS);
+			let reward_rate = Perbill::from_percent(REWARD_PERCENTAGE);
```

Comment:
```suggestion
			let reward_rate = Perbill::from_percent(0);
```
PBLO stakers receive only the trading fee rewards at the beginning